drivers/amazonec2/amazonec2.go

```diff
@@ -841,6 +850,44 @@ func (d *Driver) innerCreate() error {
 	return nil
 }
 
+// configureTags will add tags to the instance after
+// it has been created and transitioned into 'running'.
+func (d *Driver) configureTags(tagGroups string, instance *ec2.Instance) error {
+	tags := []*ec2.Tag{}
```

Comment:
If we have a good idea of what the length of a slice is before we initialize it, then we should include a length during initialization to avoid extra allocations.